source/engine/graphics/NcGraphicsImpl.h

```diff
@@ -1,45 +1,43 @@
 #pragma once
 
+#include "IGraphics.h"
 #include "camera/MainCamera.h"
 #include "ecs/PointLightSystem.h"
 #include "ecs/ParticleEmitterSystem.h"
-#include "Graphics.h"
 #include "graphics/NcGraphics.h"
```

Comment:
if we're going with not including the folder if it's our current folder (ie if 'graphics', then no need for "graphics/" in include statements, then we should delete "graphics" in "graphics/NcGraphics.h"

Reply:
NcGraphics.h is not a sibling to this file (its at include/graphics/NcGraphics.h), so this is necessary.